apps/news/news.arc

```diff
@@ -18,10 +12,13 @@
    site-color*   (color 180 180 180)
    border-color* (color 180 180 180)
    prefer-url*   t
-   newsdir*  (+ srvdir* "news/")
-   storydir* (+ srvdir* "news/story/")
-   profdir*  (+ srvdir* "news/profile/")
-   votedir*  (+ srvdir* "news/vote/")
+
+   newsdir*   (canonical-path-ts "apps/news/www/news")
```

Comment:
It looks like you're moving the news/ data directory to inside the apps/ *code* directory. Was that deliberate? Off-hand it seems like a better idea to keep them separate, not move the news/ directory.

Reply:
It was always planned to be like that, to keep the behavior of the app server and news the same. srvdir* is a relative path ("www/") that just happened to work, but since it's a relative path it can copy data to the wrong location depending on where it's used, so this just makes that location explicit.

Also, the path set in (canonical-path) always starts from the anarki directory, so the actual paths are all still the same.

Reply:
Do you mean still the same as before? When I run apps/news/news.arc without this PR the www/ directory is created under the top-level `anarki/`. With this PR it's at `apps/news/www`.

I just ran your PR, BTW, and the css loads great now. Nice work! This question of the right place for the directory isn't that important. This PR definitely improves things. So feel free to merge it.

Reply:
I think that was actually a mistake - everything was supposed to run inside /news.